index.js

```diff
@@ -94,6 +94,7 @@ class ResultStore {
 
         // anything else is an arbitrary key/val to store
         for (const key in obj) {
+            if (key.match(/^human(_html)?$/)) continue;
```

Comment:
this is redundant. `human` and `human_html` are in `all_opts` and will get skipped by the very next line.